Cubical/Categories/Instances/ZFunctors.agda

```diff
@@ -189,42 +193,42 @@ module AdjBij where
     ♭♯Id : ∀ (α  : X ⇒ Sp .F-ob A) → ((α ♯) ♭) ≡ α
     ♭♯Id _ = makeNatTransPath (funExt₂ λ _ _ → RingHom≡ (funExt (λ _ → refl)))
 
-    ♯♭Id : ∀ (φ : CommRingHom A (Γ .F-ob X)) → ((φ ♭) ♯) ≡ φ
+    ♯♭Id : ∀ (φ : CommRingHom A (𝓞 .F-ob X)) → ((φ ♭) ♯) ≡ φ
     ♯♭Id _ = RingHom≡ (funExt λ _ → makeNatTransPath (funExt₂ λ _ _ → refl))
 
-  Γ⊣SpIso : {A : CommRing ℓ} {X : ℤFunctor {ℓ}}
-         → Iso (CommRingHom A (Γ .F-ob X)) (X ⇒ Sp .F-ob A)
-  fun Γ⊣SpIso = _♭
-  inv Γ⊣SpIso = _♯
-  rightInv Γ⊣SpIso = ♭♯Id
-  leftInv Γ⊣SpIso = ♯♭Id
+  𝓞⊣SpIso : {A : CommRing ℓ} {X : ℤFunctor {ℓ}}
```

Comment:
`𝓞⊣Sp` is a *relative* adjunction (relative to the lift), see: https://ncatlab.org/nlab/show/relative+adjoint+functor